### ISSUE TITLE
Please correctly cite MMSFormer in your paper

### ISSUE THREAD

**Issue by kaykobad** [LiBingyu01/StitchFusion-StitchFusion-Weaving-Any-Visual-Modalities-to-Enhance-Multimodal-Semantic-Segmentation#2]
I appreciate that you cited and compared with my work **MMSFormer** in your paper. However, you are citing it incorrectly and an older version of the paper.

The MMSFormer paper is accepted in [OJSP](https://ieeexplore.ieee.org/document/10502124) and the title is slightly updated. 
**Old title:** Multimodal Transformer for Material Segmentation
**New Title and Link:** [MMSFormer: Multimodal Transformer for Material and Semantic Segmentation](https://ieeexplore.ieee.org/document/10502124) 

Can you please cite the paper properly in your next updated version of the paper?

Thanks!

**Comment by LiBingyu01** [comment]
Yes, I have received the message and I will update the citation information in the next version of the paper.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "LiBingyu01/StitchFusion-StitchFusion-Weaving-Any-Visual-Modalities-to-Enhance-Multimodal-Semantic-Segmentation"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月21日(星期六) 凌晨3:23
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[LiBingyu01/StitchFusion-StitchFusion-Weaving-Any-Visual-Modalities-to-Enhance-Multimodal-Semantic-Segmentation] Please correctly cite MMSFormer in your paper (Issue #2)





 
I appreciate that you cited and compared with my work MMSFormer in your paper. However, you are citing it incorrectly and an older version of the paper.
 
The MMSFormer paper is accepted in OJSP and the title is slightly updated.
 Old title: Multimodal Transformer for Material Segmentation
 New Title and Link: MMSFormer: Multimodal Transformer for Material and Semantic Segmentation
 
Can you please cite the paper properly in your next updated version of the paper?
 
Thanks!

**Comment by kaykobad** [comment]
Thank you so much
